flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/batch/sql/TimeTravelTest.java

```diff
@@ -133,7 +133,7 @@ void testTimeTravelWithUnsupportedExpression() {
                                                         + "    t1 FOR SYSTEM_TIME AS OF TO_TIMESTAMP_LTZ (0, 3)"))
                 .isInstanceOf(ValidationException.class)
                 .hasMessageContaining(
-                        "Unsupported time travel expression: TO_TIMESTAMP_LTZ(0, 3) for the expression can not be reduced to a constant by Flink.");
+                        "SQL validation failed. Unsupported time travel expression: TO_TIMESTAMP_LTZ(0, 3) for the expression can not be reduced to a constant by Flink.");
```

Comment:
Modified tests to reflect the correct exception message, after cleaning-up the legacy scala tech stack.